products/BellHybrid/apps/application-bell-relaxation/widgets/RelaxationPlayer.cpp

```diff
@@ -38,8 +38,7 @@ namespace app::relaxation
 
         auto fadeParams = audio::FadeParams{.mode = getFadeMode(), .playbackDuration = playbackDuration};
         audioModel.setPlaybackFinishedCb(std::move(onPlayerFinished));
-        audioModel.play(
-            filePath, Type::Multimedia, playbackMode, std::move(stateChangeCallback), std::move(fadeParams));
+        audioModel.play(filePath, Type::Multimedia, playbackMode, std::move(stateChangeCallback), fadeParams);
```

Comment:
why was `std::move` removed for `fadeParams`?

Reply:
This is a stack-allocated POD struct, so it cannot be moved, because there's no concept of ownership for such objects (the object is itself the data, it doesn't hold any pointers to heap-allocated data that could be moved). The only way to transfer such object is to just copy it, unless it has an explicitly defined move constructor...but then it's not an POD object anymore :wink:

It's very minor change, but I thought I'd make it, especially since `clang-tidy` complained about it too